cmd/controller-manager/main.go

```diff
@@ -115,6 +115,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "PodChaos")
 		os.Exit(1)
 	}
+	if err = (&chaosmeshv1alpha1.PodChaos{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
Can we just use a webhook for all Chaos object?  like: https://github.com/pingcap/chaos-mesh/blob/master/pkg/webhook/validation/validate.go

Reply:
But this way seems better. Should we modify the validate webhook into this form?

Reply:
@YangKeao Your proposal is better.

Reply:
> @cwen0 But this way seems better (though it has so many files and duplicate codes). Should we modify the validate webhook into this form by implementing `admission.Validator` for them?

@YangKeao agree with you.  On this issue #291 , I want to refactoring the validate webhook by implementing `admission.Validator`

So many files and duplicate codes is autogenerated by kubebuilder. I think we can just follow kubebuilder‘s rules.

Reply:
@Gallardot LGTM